storage/src/adb/any.rs

```diff
@@ -535,26 +538,30 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Array, H: CHasher, T: Translato
 
     /// Commit any pending operations to the db, ensuring they are persisted to disk & recoverable
     /// upon return from this function. Also raises the inactivity floor according to the schedule,
-    /// and prunes those operations below it.
-    pub async fn commit(&mut self) -> Result<(), Error> {
+    /// and prunes those operations below it. Batch operations will be parallelized if a thread pool
+    /// is provided.
+    pub async fn commit(&mut self, pool: Option<&mut ThreadPool>) -> Result<(), Error> {
```

Comment:
I think it would be worth:
* Exporting a `ThreadPool` type from `runtime` that is really `Arc<Rayon::ThreadPool>`.
* Providing `runtime::ThreadPool` during `init` (and cloning if it must be provided to multiple layers).

Reply:
(I suppose this won't work if `pool.install` requires mutable access)